python/hsfs/util.py

```diff
@@ -220,6 +221,28 @@ def get_hostname_replaced_url(sub_path: str):
     return url_parsed.geturl()
 
 
+def verify_attribute_key_names(attribute_name, provided_names, features):
```

Comment:
Same comment as for the java method. The method has all the code but needs to be called several times to fully validate the feature group.